docs/guides/applications/configuration-management/ansible/using-the-linode-ansible-collection-to-deploy-infrastructure/index.md

```diff
@@ -1,143 +1,180 @@
 
-        sudo ansible-galaxy collection install linode.cloud
+1.  Download the latest version of the Linode Ansible collection using the `ansible-galaxy` command:
 
-    Once the collection is installed, all configuration files will stored in the default collections folder, within the following path `~/.ansible/collections/ansible_collections/`.
+        ansible-galaxy collection install linode.cloud
 
-1.  Install the python module dependencies required for the Linode Ansible collection using pip against the `requirements.txt` file stored within the Linode collection's installation directory:
+    Once the collection is installed, all configuration files are stored in the default `~/.ansible/collections/ansible_collections/` collections folder.
 
-        pip install -r ~/.ansible/collections/ansible_collections/linode/cloud/requirements.txt
+1.  Install the Python module dependencies required for the Linode Ansible collection. The Linode collection's installation directory contains a `requirements.txt` file that lists the Python dependencies, including the official [Python library for the Linode API v4](https://github.com/linode/linode_api4-python). Use pip to install these dependencies:
 
-The Linode Ansible Collection will now be installed and ready to use to deploy and Manage Linode services.
+        sudo pip3 install -r ~/.ansible/collections/ansible_collections/linode/cloud/requirements.txt
 
-## Configuring Ansible
+The Linode Ansible collection is now installed and ready to deploy and manage Linode services.
 
-While there are many ways to leverage the Linode Ansible Collection, it is generally a good practice to use variables to securely store sensitive strings called upon later in playbooks. As the Linode collection is powered by the [Linode APIv4](https://www.linode.com/docs/api/) and deploying infrastructure generally requires the use of Secure Passwords, these variables will include the [Linode API Access token](https://www.linode.com/docs/products/tools/linode-api/guides/get-access-token/) generated in an earlier step, and a password,both encrypted at rest with [Ansible Vault](https://docs.ansible.com/ansible/latest/user_guide/vault.html).
+## Configure Ansible
 
-### Creating the Ansible Configuration File
+When interfacing with the Linode Ansible collection, it is generally good practice to use variables to securely store sensitive strings like API tokens. This section shows how to securely store and access the [Linode API Access token](https://www.linode.com/docs/products/tools/linode-api/guides/get-access-token/) (generated in the [Before You Begin](#before-you-begin) section) along with a root password that is assigned to new Linode instances. Both of these are encrypted with [Ansible Vault](https://docs.ansible.com/ansible/latest/user_guide/vault.html).
 
-Create a configuration file for Ansible that will contain the path to a file that will handle secure access to Ansible Vault.
+### Create an Ansible Vault Password File
```

Comment:
Split the previous "Creating the Ansible Configuration File" section into:

- Create an Ansible Vault Password File
- Create an Ansible Configuration File